.gitignore

```diff
@@ -6,12 +6,11 @@
 /local-config-extensions.php
 /sql-dump-*.sql
 /db-sync
-/content
```

Comment:
We actually need this directory to exist so we add a dedicated `.gitignore` inside the `content` directory which excludes everything but itself.

Reply:
This will conflict with anyone with a checked-out repository inside `content`, and is a blocker.